FLEDGE.md

```diff
@@ -676,14 +699,17 @@ The arguments to this function are:
       'componentSeller': 'https://www.some-other-ssp.com',
       'interestGroupOwner': 'https://www.example-dsp.com/',
       'renderURL': 'https://cdn.com/url-of-winning-creative.wbn',
-      'bid:' bidValue,
+      'bid': bidValue,
+      'bidCurrency': ...,
```

Comment:
```suggestion
      'bidCurrency': 'USD',
```
I feel like we should put something more explicit here.  I don't feel strongly though, I was just thinking it might help folks understand the type of the data.

Reply:
Done